src/utils/webgl.js

```diff
@@ -34,3 +34,8 @@ export function patchWebGLRenderingContext() {
     };
   }
 }
+
+export function getWebGLVersion(renderer) {
+  const gl = renderer.getContext("webgl") || renderer.getContext("experimental-webgl");
```

Comment:
I don't think Three.js `WebGLRenderer.getContext()` takes an argument.

https://threejs.org/docs/#api/en/renderers/WebGLRenderer.getContext

```suggestion
  const gl = renderer.getContext();
```

Reply:
Ops yep, fixed, thanks!